src/main/java/com/nylas/Neural.java

```diff
@@ -0,0 +1,105 @@
+package com.nylas;
+
+import okhttp3.HttpUrl;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Neural {
+	protected final String neuralPath = "neural";
+	protected final NylasAccount account;
+	protected final NylasClient client;
+	protected final String accessToken;
```

Comment:
I think these fields ought to be `private` as they are only used internally.  Or is there a reason to choose `protected`?

Reply:
nope, was supposed to be private -- pretty sure I autofilled wrong. That's on me, good catch.